_data/tools.yml

```
@@ -307,6 +307,26 @@
       url: /docs/current/connector/pinot.html
     - urltext: Trino takes a sip of Pinot! from Trino Community Broadcast 13
       url: /episodes/13.html
+- name: Apache Ranger
+  anchor: apache-ranger
+  category: add-on
+  logo: /assets/images/logos/apache-ranger.png
+  logosmall: /assets/images/logos/apache-ranger-small.png
+  description: |
+    Apache Ranger is a framework to enable, monitor and manage comprehensive
```

```suggestion
    Apache Ranger is a framework to enable, monitor, and manage comprehensive
```
leave it to me to catch the missing Oxford comma